clumpr/HistogramMainInput.py

```diff
@@ -2,38 +2,23 @@
 # Adrian Garcia, March 2020
 # Main input function for HistogramMain
 
-def getInput(filename):
+def getInput(filename, d, t, l, args):
     """ 
-    :param filename: the path to a .newick file with the input trees and tip mapping.
-    :return: dictionary of arguments where key is parameter name and value is parameter value.
+    Get additional input from user for computing and saving the PDV
+    :param filename <str> - the path to a .newick file with the input trees and tip mapping
+    :param d <float> - the cost of a duplication
+    :param t <float> - ^^ transfer
```

Comment:
I don't see the downside of writing it out as "the cost of transfer" and "the cost of lost" . Also, I thought we have mostly used an int instead of a float. I might be wrong though. :3